core/src/main/java/net/sourceforge/jnlp/Parser.java

```diff
@@ -831,12 +831,11 @@ public EntryPoint getEntryPointDesc(final Node parent) throws ParseException {
      * @return the applet descriptor.
      *
      * @throws ParseException if the JNLP file is invalid
-     *
-     * TODO: parse and set {@link AppletDesc#getProgressClass()}
      */
-    private AppletDesc getApplet(final Node node) throws ParseException {
+    AppletDesc getAppletDesc(final Node node) throws ParseException {
         final String name = getRequiredAttribute(node, AppletDesc.NAME_ATTRIBUTE, R("PUnknownApplet"), strict);
         final String main = getMainClass(node, true);
+        final String progressClass = getAttribute(node, ApplicationDesc.PROGRESS_CLASS_ATTRIBUTE, null);
```

Comment:
I assume this should be AppletDesc.PROGRESS_CLASS_ATTRIBUTE

Reply:
Thanks for seeing this, fixed.